generators/chipyard/src/main/scala/unittest/TestHarness.scala

```diff
@@ -0,0 +1,9 @@
+package chipyard.unittest
+
+import chisel3._
+import freechips.rocketchip.config.Parameters
+
+class TestHarness(implicit val p: Parameters) extends Module {
+  val io = IO(new Bundle { val success = Output(Bool()) })
+  io.success := Module(new UnitTestSuite).io.finished
```

Comment:
Can't this be done by setting `BuildSystem` to `Module(new UnitTestSuite)`, and using a `IOBinder` to connect `io.success` to `dut.io.finished`?

Reply:
But then I'd need to change the configuration inside testchipip and have it depend on chipyard or create one inside chipyard.